components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -1946,8 +1955,10 @@ def removeWrapperAndNullableTypes(types):
                 if parentName:
                     descriptor = descriptorProvider.getDescriptor(parentName)
                     extras += [descriptor.path, descriptor.bindingPath]
-            elif t.isType() and t.isMozMap():
+            elif t.isType() and t.isRecord():
                 extras += ['dom::bindings::mozmap::MozMap']
+            elif isinstance(t, IDLPromiseType):
```

Comment:
Not all `t` have an `isPromise` method, probably an error but I don't want to have to maintain yet another `WebIDL.py` local patch for that.